modules/mbedtls/Kconfig.psa

```diff
@@ -301,7 +301,7 @@ config PSA_WANT_KEY_TYPE_RSA_PUBLIC_KEY
 	default y if PSA_CRYPTO_ENABLE_ALL
 
 config PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_BASIC
-	bool "PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_BASIC" if !MBEDTLS_PROMPTLESS
+	bool
 	default y if PSA_CRYPTO_ENABLE_ALL
```

Comment:
Remove those `PSA_WANT_KEY_TYPE_*_KEY_PAIR_BASIC` completely from this file as you are adding them to `Kconfig.psa.logic`?